composer.json

```diff
@@ -1,10 +1,25 @@
 {
-  "name": "drupal/dvf",
-  "description": "Data Visualisation Framework makes it easy to generate visualisations from different data sources.",
-  "type": "drupal-module",
-  "license": "GPL-2.0+",
-  "minimum-stability": "dev",
-  "require": {
-    "drupal/core": "~8"
-  }
+    "name": "drupal/dvf",
+    "description": "Data Visualisation Framework makes it easy to generate visualisations from different data sources.",
+    "type": "drupal-module",
+    "homepage": "https://github.com/govCMS/dvf",
+    "authors": [
+        {
+            "name": "Jorge Castro",
+            "homepage": "https://www.drupal.org/u/jorgegc"
+        },
+        {
```

Comment:
probs should namecheck @jez500 here...

Reply:
Thank you Toby, I agree :)

Reply:
I thought you guys didn't like @jez500 and that is why his name was not included in the PR I copied this from :-) Either that or he didn't want the fame and glory that come with maintaining an awesome module like DVF.

Reply:
This has been [fixed](https://github.com/govCMS/dvf/blob/feature/dvf-json/composer.json#L7-L10).